homework/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 {
+  let repositoryList = [];
```

Comment:
Don't use global variables like this. It makes it difficult to find out where in your code references are made to it.

Reply:
done